docs/reference/announcements.md

```diff
@@ -30,9 +30,13 @@ Note that the actual values shown in this screenshot don't correspond to any act
 
 ### Removal of Web Modeler's beta API
 
-As of the 8.5 release, the Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
+The Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
 For a migration guide, see the [Web Modeler API documentation](/apis-tools/web-modeler-api/index.md#migrating-from-beta-to-v1).
 
+### Updated SaaS URLs
+
+The SaaS URL is being shortened from cloud.camunda.io to camunda.io. Both URLs will be available until 8.6, after which point there will be an auto-redirect to the shorter URLs. If you have internal whitelisting or active rules for cloud.camunda.io, please transition them to camunda.io. This change primarily affects Console and Modeler. During signup, users are briefly redirected through Accounts (accounts.cloud.camunda.io), which will also be updated.
```

Comment:
```suggestion
We have simplified the URL for Camunda SaaS from cloud.camunda.io (ex. [console.cloud.camunda.io](https://console.cloud.camunda.io/)) to camunda.io (ex. [console.camunda.io](http://console.camunda.io/)).

Users will be directed to the new URLs.  Both URLs will continue to be active for at least 18 months so navigation from supported versions of web apps like Operate is still possible. 

Internal whitelisting or active rules for [cloud.camunda.io](http://cloud.camunda.io/) must be transitioned to the new [camunda.io](http://camunda.io/) URL. This change primarily affects Console and Modeler. During signup, users are briefly redirected through [accounts.cloud.camunda.io](http://accounts.camunda.io/), which will also be updated.```

Reply:
@HanselIdes, should we mention that the change will be coming for all components around 9th of July like in the document you shared? For me, it seems that it is already live for all components.

Reply:
@JacobiCamunda Good catch - for some reason I assumed we would send out the announcement alongside the July release. I've updated the text recommendation